easydata.js/packs/core/src/data/data_column.ts

```diff
@@ -23,6 +23,7 @@ export interface DataColumnDescriptor {
     dfmt?: string;
     gfct?: string;
     style?: DataColumnStyle;
+    description?: string;
```

Comment:
It's usually better to not use `?` in interface declaration because it gets confusing very soon of in which situations the column has a value and in which it does not.
I don't suggest changing the previous code, but `description: string | null;` is much better.